.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This file(or the containing folder) needs to be added to the .gitignore and no longer tracked

Reply:
And the other files in the .vscode folder